evm_loader/cli/src/main.rs

```diff
@@ -1022,6 +1131,15 @@ fn main() {
                         .validator(is_valid_hexdata)
                         .help("Transaction data")
                 )
+                .arg(
+                    Arg::with_name("value")
+                        .value_name("VALUE")
+                        .takes_value(true)
+                        .index(4)
+                        .required(false)
+                        .validator(is_amount_u256)
+                        .help("Transaction value")
+                )
```

Comment:
You have 2 optional parameters. How neon-cli behaves when one/all of them are not present?
Write a test. Without data. Without value. Without data and value.

Reply:
Added tests
https://buildkite.com/cyberway/neon-proxy/builds/868#12312627-795e-422e-b14f-8eeb9a2b08f7/22-105
https://buildkite.com/cyberway/neon-proxy/builds/868#12312627-795e-422e-b14f-8eeb9a2b08f7/22-143
https://buildkite.com/cyberway/neon-proxy/builds/868#12312627-795e-422e-b14f-8eeb9a2b08f7/22-179

Reply:
You added tests for proxy. Where are the neon-cli tests?
How this will be handled?
`neon-cli emulate 0x0000000000000000000000000000000000000000 0x0000000000000000000000000000000000000000 0x42`
no data
0x42 - transfer value

Reply:
It is wrong use of neon-cli 
You can see help

```
neon-cli emulate --help
...
<DATA>        Transaction data or 'None'
```

Reply:
`data` is an optional parameter.
If you want it always present set `required(true)`

Reply:
Also why 'None' starts with big 'N' and all other keywords are in lower_snake_case?
Use case-insensitive string comparison